src/web/jcc/client/components/RCControllerPanel.tsx

```diff
@@ -147,21 +194,21 @@ export default class RCControllerPanel extends React.Component {
     }
 
     updateRudderDirectionMove(event: IJoystickUpdateEvent) {
```

Comment:
We should add a comment for this function since it was changed in this pull request.